.eslintrc

```diff
@@ -35,6 +35,18 @@
     "es6": true
   },
 
+  "globals": {
```

Comment:
I was thinking the next two things required on the project were linting as part of the build process, as well as adding CI.

In my grunt-based projects, I usually set-up various targets (client source, node code, tests) with slightly different setups. We probably can achieve the same result with gulp though I am no expert.

Should we keep this in a separate requirement?